ui/app/templates/csi/plugins/plugin/index.hbs

```diff
@@ -0,0 +1,191 @@
+            </div>
+          </div>
+        </div>
+      </div>
+    {{/if}}
+    <div class="column">
+      <div class="boxed-section" data-test-plugin-node-availability>
+        <div class="boxed-section-head is-hollow">Node Health</div>
+        <div class="boxed-section-body">
+          <div class="columns is-centered is-bottom-aligned {{unless model.controllerRequired "is-max-half"}}">
+            <div class="column is-half">
+              {{gauge-chart
+                label="Availability"
+                value=model.nodesHealthy
+                total=model.nodesExpected}}
+            </div>
+            <div class="column">
+              <div class="metric">
+                <h3 class="label">Available</h3>
+                <p class="value">{{model.nodesHealthy}}</p>
+              </div>
+            </div>
+            <div class="column">
+              <div class="metric">
+                <h3 class="label">Expected</h3>
+                <p class="value">{{model.nodesExpected}}</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  {{#if model.controllerRequired}}
+    <div class="boxed-section" data-test-controller-allocations>
+      <div class="boxed-section-head">
+        Controller Allocations
+      </div>
+      <div class="boxed-section-body {{if model.controllers "is-full-bleed"}}">
+        {{#if model.controllers}}
+          {{#list-table
+            source=(take 10 sortedControllers)
+            class="with-foot" as |t|}}
+            {{#t.head}}
+              <th class="is-narrow"></th>
+              <td>ID</td>
+              <th>Created</th>
+              <th>Modified</th>
+              <th>Health</th>
+              <th>Client</th>
+              <th>Job</th>
+              <th>Version</th>
+              <th>Volumes</th>
+              <th>CPU</th>
+              <th>Memory</th>
+            {{/t.head}}
+            {{#t.body key="model.allocID" as |row|}}
+              {{plugin-allocation-row
+                data-test-controller-allocation=row.model.allocID
+                pluginAllocation=row.model}}
+            {{/t.body}}
+          {{/list-table}}
+        {{else}}
+          <div class="empty-message" data-test-empty-controller-allocations>
+            <h3 class="empty-message-headline" data-test-empty-controller-allocations-headline>No Controller Plugin Allocations</h3>
+            <p class="empty-message-body" data-test-empty-controller-allocations-message>No allocations are providing controller plugin service.</p>
+          </div>
+        {{/if}}
+      </div>
+      {{#if model.controllers}}
+        <div class="boxed-section-foot">
+          <p class="pull-right">
+          {{#link-to "csi.plugins.plugin.allocations"
+            model
+            data-test-go-to-controller-allocations
+            (query-params qpType=(qp-serialize (array "controller")))}}
+            View All Controller Allocations
```

Comment:
Also about this realm, I wondered whether it might be useful to somehow indicate whether truncation is actually occurring. I have on occasion been annoyed at interfaces that have “show all”-type features but don’t let me know whether there’s actually more to show. This could say “View All X Controller Allocations” for instance… clunky though 😐

Reply:
I can do that. Changing it like you suggest actually makes it consistent with how we treat recent allocations on the job detail page.